app/templates/views/service-settings/email-reply-to/_verify-updates.html

```diff
@@ -29,7 +29,7 @@ <h2 class='banner-title' data-module="track-error" data-error-type="reply-to-ema
           There’s a problem with your reply-to address
         </h2>
         <p>
-          We sent an email to ‘{{ reply_to_email_address }}’ but it wasn’t delivered.
+          We sent an email to ‘{{ reply_to_email_address }}’ but it cannot be delivered.
```

Comment:
Feel like this should be past tense, for example

```suggestion
          We sent an email to ‘{{ reply_to_email_address }}’ but it was not delivered.
```
or
```suggestion
          We sent an email to ‘{{ reply_to_email_address }}’ but it could not be delivered.
```